README.md

```diff
@@ -42,6 +42,12 @@ Add two dependencies in the application ```build.gradle```:
     implementation 'com.veriff:veriff-library:2.3.4'
     implementation 'io.probity.sdk:collector:1.0.0'
 ```
+
+#### Permissions
+
+The SDK requests for all the permissions it needs, please make sure that the CAMERA, RECORD_AUDIO and WRITE_EXTERNAL_STORAGE permissions are
```

Comment:
the first part of the sentence reads weird, maybe reword it as `The SDK will request all the permissions it needs, `